requirements-dev.txt

```diff
@@ -12,5 +12,5 @@ responses==0.25.0
 # mypy and PEP 561 stubs
 mypy==1.9.0
 django-stubs==4.2.7
-djangorestframework-stubs==3.14.2
-types-requests==2.31.0.20240406
+djangorestframework-stubs==3.14.5
+types-requests==2.31.0.9
```

Comment:
This should be done in a separate PR.

Reply:
Reverted in 946871f9cd95aabc19823ed641c37ba54481d3a2.

Reply:
It was not removed 🤔

Reply:
There was a newline issue I missed. I fixed it in 55e336ed119238aa8d9d01d3015956ba095f76ea.

Reply:
Sorry yes, it was the newline. The diff confused me 😬